packages/default-storage/package.json

```diff
@@ -8,9 +8,9 @@
   "types": "lib/typescript/index.d.ts",
   "files": [
     "RNCAsyncStorage.podspec",
-    "android/build.gradle",
-    "android/src",
-    "android/testresults.gradle",
+    "android/",
+    "!android/.gradle",
+    "!android/build",
```

Comment:
This might be related to the rewrite of the glob implementation in npm 9. We've hit issues with it as well: https://github.com/npm/npm-packlist/issues/152

Reply:
not cool!